lib/logidze/versioned_association.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+module Logidze
+  module VersionedAssociation
+    def load_target
+      target = super
+
+      return target if inversed
```

Comment:
Yep. But this `post` object would exactly the object we call association on, so it will be at the _correct_ time. But, nevertheless, we need specs.